app.js

```diff
@@ -15,6 +15,12 @@ app.get('/api/', getApiEndPoints)
 
 app.get('/api/articles/:article_id', getArticlesById);
 
+app.get('/api/articles', getAllArticles);
+
+app.all('/api/*', (req, res, next) => {
```

Comment:
Nice work having this generic bad path handler in here. What if I navigate to the endpoint /banana? A bad path may not just start with /api so best to simply pass '*' into here to grab all bad paths.

Reply:
removed /api/ that leaves * only.